src/components/TabsButtons/TabsButtons.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import classNames from 'classnames';
+import './TabsButtons.scss';
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: string,
+  onClick(title: string, id: string, content: string): void,
+};
+
+export const TabsButtons: React.FC<Props> = ({ tabs, selectedTab, onClick }) => (
+  <div className="tabs__buttons">
+    {
+      tabs.map(tab => (
+        <button
+          key={tab.id}
+          id={tab.id}
+          className={classNames(
+            'tabs__button',
+            {
+              'tabs__button--active': tab.id === selectedTab,
+            },
+          )}
+          type="button"
```

Comment:
Default value

Reply:
I have an error without this(
I think we should always specify the type for the button